tests/test_application.py

```diff
@@ -2,7 +2,6 @@
 from unittest import mock
 
 import pytest
-
```

Comment:
Can you add this back, please 

```
Imports should be grouped in the following order:

standard library imports
related third party imports
local application/library specific imports
You should put a blank line between each group of imports.
```

 - http://legacy.python.org/dev/peps/pep-0008/#imports

Reply:
Done